ironfish/src/network/peers/peerManager.test.ts

```diff
@@ -516,7 +532,7 @@ describe('PeerManager', () => {
       )
     })
 
-    it('Does not create a connection if Peer has disconnectUntil set', () => {
+    it('canConnectToWebSocket returns false if Peer has disconnectUntil set', () => {
```

Comment:
Can you revert changes to this test? I think it should be fine again to call `connectToWebSocket` instead of testing `canConnectToWebSocket`